package.json

```diff
@@ -1,23 +1,25 @@
 {
-  "name": "how",
-  "version": "0.1.0",
-  "description": "Ask a cli tool about other cli tools.",
-  "module": "./src/index.ts",
-  "type": "module",
-  "scripts": {
-    "how": "bun run ./src/index.ts --",
-    "compile": "bun build ./src/index.ts --compile --minify --sourcemap --outfile how && chmod +x ./how",
-    "replace-bin": "sudo mv ./how /usr/local/bin/how"
-  },
-  "dependencies": {
-    "@inquirer/prompts": "^5.1.2",
-    "chalk": "^5.3.0",
-    "clipboardy": "^4.0.0"
-  },
-  "devDependencies": {
-    "@types/bun": "latest"
-  },
-  "peerDependencies": {
-    "typescript": "^5.0.0"
-  }
+	"name": "how",
+	"version": "0.1.0",
+	"description": "Ask a cli tool about other cli tools.",
+	"module": "./src/index.ts",
+	"type": "module",
+	"scripts": {
+		"how": "bun run ./src/index.ts --",
+		"compile": "bun build ./src/index.ts --compile --minify --sourcemap --outfile how",
+		"check": "biome check ./src --fix"
+	},
+	"dependencies": {
+		"@inquirer/prompts": "^5.1.2",
+		"chalk": "^5.3.0",
+		"clipboardy": "^4.0.0",
+		"env-paths": "^3.0.0"
```

Comment:
bun can read `.env` files out of the box. 

and i only needed `.env` to store my testing keys and that's why i added it in gitignore, they are not bundled with executable, they only for development.

Reply:
You seem to be uninformed on what that package is for – it's for getting standard configuration directories (i.e. `$HOME/.config/how`)

Reply:
i don't see why we need that package then.
i want to store config inside `$HOME/.how/config.json` path. i don't need external package for that.

Reply:
First, applications should always use a standard directory. Storing your configuration in the home directory in its own dotfolder is considered very bad practice. Second, you are not considering people with filesystems that don't follow modern Unix like paths.

Reply:
if it's in home it's easy to find and delete. a lot of tools locate themselves in home and i find it very nice.
<img width="288" alt="Screenshot 2024-07-12 at 17 27 37" src="https://github.com/user-attachments/assets/094947b5-a316-4981-8148-51efc09498e6">

i don't plan to support non-unix systems. i made this tool for me and my friends. you are free to fork and make any changes you want. 

you should question your premise on "best practices"

10 years ago jquery and prototype inheritance was considered a "best practice"

Reply:
There's a reason XDG standard directories exist. It makes your system far easier to back up and makes application configuration far simpler.